Modules/pyexpat.c

```diff
@@ -703,6 +703,37 @@ get_parse_result(pyexpat_state *state, xmlparseobject *self, int rv)
 
 #define MAX_CHUNK_SIZE (1 << 20)
 
+/*[clinic input]
+pyexpat.xmlparser.SetReparseDeferralEnabled
+
+    cls: defining_class
+    enabled: bool
+    /
+
+Enable/Disable reparse deferral; enabled by default since Expat 2.6.0.
+
+Large tokens may require many parse calls before enough data is available
+for Expat to parse it in full. If Expat retried parsing the token
+on every parse call, parsing could take quadratic time. To avoid this,
+Expat only retries once a significant amount of new data is available.
+This function allows disabling this behavior.
```

Comment:
It should be in the module documentation. The first line is enough for docstring.

Reply:
@serhiy-storchaka alright, dropped with next push, re-adding elsewhere once we're entering the second phase on documentation (as suggested above).